cmd/soroban-rpc/internal/db/migrations/02_transactions.sql

```diff
@@ -0,0 +1,16 @@
+-- +migrate Up
+
+-- index to find transactions in ledgers by hash
+CREATE TABLE transactions (
+    hash NCHAR(64) PRIMARY KEY,
+    ledger_sequence INTEGER NOT NULL,
+    application_order INTEGER NOT NULL,
+    -- possibly a useful search index for Soroban-oriented queries, e.g.
+    -- transactions that involve the SAC or have just events in general
+    is_soroban BOOLEAN NOT NULL,
```

Comment:
I think it's better to remove this column unless there is an immediate need. BTW, there is no index on this column so it won't be efficient to filter by `is_soroban`. If in the future, we need to implement an indexing feature for soroban transactions we can always add this column and an index in a subsequent migration. Given that soroban-rpc has a limited retention window, it isn't as expensive to reingest compared to horizon

Reply:
> it isn't as expensive to reingest compared to horizon

Yeahh the idea was that it was still quite expensive to do so (vs. just a migration), being easier than Horizon is kind of a low bar heh, and I felt like it could be valuable to add in advance (I think we'll want it for `getEvents`), but you're right that maybe it's premature - dropped the column :+1: